Http/frontendRoutes.php

```diff
@@ -4,8 +4,8 @@
 
 /** @var Router $router */
 
-$router->group(['prefix' => 'blog'], function (Router $router) {
+$router->group(['prefix' => trans('blog::settings.blog-slug')], function (Router $router) {
     $locale = LaravelLocalization::setLocale() ?: App::getLocale();
-    $router->get('posts', ['as' => $locale . '.blog', 'uses' => 'PublicController@index']);
-    $router->get('posts/{slug}', ['as' => $locale . '.blog.slug', 'uses' => 'PublicController@show']);
+    $router->get(trans('blog::settings.posts-slug'), ['as' => $locale . '.blog', 'uses' => 'PublicController@index']);
```

Comment:
While we're at it, can we get rid of the `$locale` in the route names? It doesn't appear to do anything at all.